src/main/java/doit/jpastudy2/repository/School.java

```diff
@@ -0,0 +1,36 @@
+package doit.jpastudy2.repository;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class School {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "School_id")
+    private Long id;
+
+    @Column(name = "School_name")
+    private String schoolName;
+
+    @Column(name = "Total_students")
+    private int totalStudents;
+
+    @Column(name = "Average_grade")
+    private double averageGrade;
```

Comment:
잘 작성해주셨습니다!
틀린 것은 아니지만! 관례상 자주 사용되는 패턴이 있습니다.
MySQL에서는 Column 이름을 snake_case로 작성하는 것이 일반적입니다.
실제로도 `@Column(name = ~)`를 생략하게 되면, 이를 snake_case로 변경하여 DB에 생성합니다.
그리고 table명_id 구조로 PK를 작성하는 편이기도 합니다.
따라서 아래처럼 써보시는 것이 어떠할까 싶네용

```suggestion
    @Column(name = "school_id")
    private Long id;

    @Column(name = "school_name")
    private String schoolName;

    @Column(name = "total_students")
    private int totalStudents;

    @Column(name = "average_grade")
    private double averageGrade;
```